Baby Monitor/Source Files/Services/Websocket/WebSocketProtocol.swift

```diff
@@ -43,14 +43,27 @@ final class PSWebSocketWrapper: NSObject, WebSocketProtocol {
                 return false
             }
         }
+        
+        func toWebSocketConnectionStatus() -> WebSocketConnectionStatus {
+            switch self {
+            case .connected:
+                return .connected
+            case .connecting:
+                return .connecting
+            default:
```

Comment:
I would suggest to list here other cases instead of default.

Reply:
ok